lib/service/translation/amazon_translate.dart

```diff
@@ -0,0 +1,314 @@
+import 'dart:convert';
+
+import 'package:crypto/crypto.dart';
+import 'package:convert/convert.dart';
+import 'package:flutter_arb_translator/models/api_result.dart';
```

Comment:
please use relative paths for in-project references:
`import '../../models/api_result.dart';`